2023-07-14-sketch_map_tool_new_version.md

```diff
@@ -0,0 +1,86 @@
+
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog-20230714_map.png">
+<p class="caption">Sketch Map of Risk Areas from a project in Colombia (Letzner 2023)</p>
+</figure>
+
+## Selected use cases of the Sketch Map Tool
+
+The Sketch Map Tool can be used by local stakeholders to engage communities in gathering information about their experience and perception of previous flood events or other hazards in their neighborhoods. In the past, the tool has been used by various actors in risk assessments, to design risk reduction measures, or to enhance communication by increasing risk awareness among citizens.
+
+The development of the first prototypes of the Sketch Map Tool started over four years ago in the [Waterproofing data project](https://www.geog.uni-heidelberg.de/gis/waterproofing.html) to support the collection of flood risk perception in two vulnerable Brazilian communities.
+
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog-20230714_group.png">
+</figure>
+
+Since then, the tool has been tested in different contexts and projects around the world including Germany, Mozambique and Madagascar. In Mozambique, for example, users recorded flood areas and regions with increased crime using the Sketch Map Tool in workshops. The collected data was used in disaster risk reduction projects by the German Red Cross and [Mozambique Red Cross Society](https://piroi.croix-rouge.fr/piroi/members-of-piroi/mozambique-red-cross-society/?lang=en).
+
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog-20230714_double.png">
+</figure>
+
+Currently, our former HeiGIT intern Kimon Letzner explores the potential and limitations of the Sketch Map Tool in the scope of his Master’s studies at Lund University. His focus is on the community mapping process in the context of the [Enhanced Vulnerability and Capacity Assessment (EVCA)](https://communityengagementhub.org/resource/evca-guide/) in the International Red Cross Red Crescent Movement based on a project with the [Colombian Red Cross National Society](https://www.cruzrojacolombiana.org/).
+
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog-20230714_community_mapping.jpeg">
+<p class="caption">Community mapping (Letzner 2023)</p>
+</figure>
+
+In May 2023, the Sketch Map tool was presented and used in a practical exercise at a regional training of the [International Federation of Red Cross and Red Crescent Societies](https://www.ifrc.org/) “Road Map to Community Resilience via Enhanced Vulnerability and Capacity Assessment (EVCA)” in Kuala Lumpur, Malaysia. A write-up of the training can be found here. This event represented another crucial step towards implementing the tool in the standard processes of the Red Cross Red Crescent Movement worldwide.
```

Comment:
text says "can be found here" but there is no link.

Reply:
Added the link. Thanks for catching that!